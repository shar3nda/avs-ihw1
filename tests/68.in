10
3 2 8 9 4 3 8 3 8 2