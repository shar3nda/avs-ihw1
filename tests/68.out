0 2 0 -2 8 10 -2 5 -2 3 -1 5 0